app/models/exam.rb

```diff
@@ -1,19 +1,28 @@
 class Exam < ApplicationRecord
+
   include GuideContainer
   include FriendlyName
-
-  validates_presence_of :start_time, :end_time
+  include TerminalNavigation
 
   belongs_to :organization
+  belongs_to :course, optional: true
 
   has_many :authorizations, class_name: 'ExamAuthorization', dependent: :destroy
   has_many :users, through: :authorizations
 
+  enum passing_criterion_type: [:none, :percentage, :passed_exercises], _prefix: :passing_criterion
```

Comment:
Why not just calling it `passing_criterion`? Is there any difference between the criterion and the criterion type?